catalog/spatial/geocoding/spatial-geocoding-plugin/src/main/java/org/codice/ddf/spatial/geocoding/plugin/GeoCoderPlugin.java

```diff
@@ -77,11 +75,11 @@ public UpdateRequest process(UpdateRequest input)
             return input;
         }
 
-        geoCoder = geoCoderFactory.getService();
+        GeoCoder geoCoder = geoCoderFactory.getService();
         input.getUpdates()
                 .stream()
                 .map(Map.Entry::getValue)
-                .forEach(this::setCountryCode);
+                .forEach(metacard -> setCountryCode(metacard, geoCoder));
```

Comment:
I think we would want a single geoCoder per batch of metacards. Otherwise we are instantiating a new one for each metacard in the request.